cji_smoke_test.sh

```diff
@@ -61,6 +61,19 @@ if [ "$IQE_SELENIUM" = "true" ]; then
     SELENIUM_ARG=" --selenium "
 fi
 
+# check if there is a iqe-tests container image tag with the corresponding PR
+# number if not use the default IQE_IMAGE_TAG tag which you have to set in
+# order for this to work properly
+if [ ! -z "$ghprbPullId" ]; then
+  export IQE_IMAGE_TAG_TMP="${IQE_IMAGE_TAG}-pr-${ghprbPullId}"
+  set +e
+  docker manifest inspect quay.io/cloudservices/iqe-tests:"$IQE_IMAGE_TAG_TMP"
+  if [[ $? -eq 0 ]]; then
+      export IQE_IMAGE_TAG="$IQE_IMAGE_TAG_TMP"
```

Comment:
Can we 'echo' a log message that we are now setting a different tag since we found an IQE plugin image related to this PR?